### DISCUSSION TITLE
Add logbook notes

### DISCUSSION THREAD

**Discussion by TheBitBanger** [nvim-orgmode/orgmode#147] [Ideas]
Hello,

I'm brand new to orgmode, found it on reddit and figured it is exactly what I've been manually doing with my markdown files but on steroids.

I hope this in the scope of orgmode.nvim, as it actually is on emacs orgmode, but I'm not entirely familiar with the roadmap for this project (it's a little vague on the README).

The idea is having the ability to add dated notes in the logbook (in this case in particular, however, we should be able to add notes everywhere, not sure how many of these should be part of this implementation), preferably after clocking out of a task.

It seems the emacs version can do it like in this [video](https://www.youtube.com/watch?v=nUvdddKZQzs), there's some [docs](https://orgmode.org/org.html#Drawers) on it but sadly there are no examples.

It is very useful when you want to keep track of long tasks, sometimes instead of making multiple super small tasks, you have a bigger one and just log work there, sometimes it makes more sense to split the task, it depends.

This is how I'm currently manually doing it:

```
* TODO Make more e-liquid
  :LOGBOOK:
  CLOCK: [2021-11-10 Wed 11:51]--[2021-11-10 Wed 12:08] => 0:17
  - Progress log on <2021-11-10 Wed 12:14>
    Made 3 more liquids, should buy a creamy/milky liquid and some fruity ones
  :END:
```

Having implemented this. the next logical step would be to create something to visualize such entries, it would be amazing to be able to do it right in the agenda, inside of an expanded clock report (for this part it would be awesome to be able to link CLOCK entries and progress logs, maybe using an optional word like PROGRESSLOG after the CLOCK), so users can toggle it on and off, but I'm aware this is probably out of the scope of orgmode and maybe better implemented as a plugin.

On the side of plugins, this is by no means a request (unless it fits in the scope of orgmode :D), being able to collect all these progress entries and display them in a clever way (progress log by (sub)tree, file, date range or any kind of expression) would lead to some kind of journal, which may even be exported into some format and become a digital journal of just about anything that could be "task-ified" or "org-ified". This can even lead to fun information such as "on this day...." and display entries from previous years.

I hope this idea is useful and at least some part of it makes it in the code. I'm not a lua developer, but learning sounds fun and a plugin is probably a very good project, so if we go in this direction, count me in!

Let me know if you have any comments.

Thanks in advance!

Best regards.

**Comment by kristijanhusak** [comment]
Added an issue to track this and implement at some point https://github.com/kristijanhusak/orgmode.nvim/issues/149
